tests/conftest.py

```diff
@@ -3,15 +3,25 @@
 import cellprofiler.object
 import cellprofiler.pipeline
 import cellprofiler.workspace
+import numpy
 import skimage.data
 import pytest
 
 
-@pytest.fixture(scope="module")
-def image():
-    example = skimage.data.camera()
-
-    return cellprofiler.image.Image(example)
+@pytest.fixture(scope="module",
+                params=[
+                    (skimage.data.camera()[0:128, 0:128], 2),
+                    (skimage.data.astronaut()[0:128, 0:128, :], 2),
+                    (numpy.tile(skimage.data.camera()[0:32, 0:32], (2, 1)).reshape(2, 32, 32), 3)
```

Comment:
I think this is fine for the moment, but we should really get a volume upstream to skimage.